vector/src/main/java/im/vector/riotx/features/crypto/verification/VerificationBottomSheetViewModel.kt

```diff
@@ -362,7 +362,9 @@ class VerificationBottomSheetViewModel @AssistedInject constructor(
                         }
                     }
                 } catch (failure: Throwable) {
-                    _viewEvents.post(VerificationBottomSheetViewEvents.ModalError(failure.localizedMessage))
+                    failure.localizedMessage?.let {
```

Comment:
same here, use a default error string and display it (R.string.unexpected_error)